cdc-accelerator/feature/acl.test.js

```diff
@@ -44,4 +44,29 @@ describe('ACLs test suite', () => {
             await expect(acls.init(aclIDs, getSiteInfo.partnerId, srcDirectory, getSiteInfo.dataCenter)).rejects.toThrow(new Error(JSON.stringify(expectedGigyaResponseNok)))
         })
     })
+    describe('Build test suit', () => {
+        test('All ACL files are built successfully', async () => {
+            const srcFileContent = JSON.stringify(expectedAclResponse.acl)
+            const aclName = Object.keys(expectedACLFileContent)
+            const buildPermissionGroupDirectory = path.join(partnerBuildDirectory, permissionGroupDirectoryName)
+            const dirExists = true
+            fs.existsSync.mockReturnValue(dirExists)
+            fs.readdirSync.mockReturnValue([`${aclName[0]}.json`, `${aclName[1]}.json`])
+
+            fs.readFileSync.mockReturnValue(srcFileContent)
+            acls.build(buildPermissionGroupDirectory)
+            expect(fs.writeFileSync.mock.calls.length).toBe(2)
+            expect(fs.writeFileSync).toHaveBeenNthCalledWith(
+                1,
+                path.join(buildPermissionGroupDirectory, `${acls.getName()}`, `${aclName[0]}.json`),
+                JSON.stringify(JSON.parse(srcFileContent)),
+            )
+
+            expect(fs.writeFileSync).toHaveBeenNthCalledWith(
+                2,
+                path.join(buildPermissionGroupDirectory, `${acls.getName()}`, `${aclName[1]}.json`),
+                JSON.stringify(JSON.parse(srcFileContent)),
```

Comment:
There is no need to call JSON.stringify(JSON.parse(, the srcFileContent has already the right format